ros2topic/ros2topic/api/__init__.py

```diff
@@ -177,11 +184,57 @@ def profile_configure_short_keys(
 
 
 def qos_profile_from_short_keys(
```

Comment:
probably we can extend test here?

https://github.com/ros2/ros2cli/blob/70d4991b66aa30d0b3b0ea9b9351e523b00d54ec/ros2topic/test/test_qos_conversions.py#L20

Reply:
extended in https://github.com/ros2/ros2cli/pull/788/commits/b341ee576b4687d8cba5b8e8b0bba5e66a3f8825